app/assets/stylesheets/modules/top_bar.scss

```diff
@@ -0,0 +1,44 @@
+@import 'variables';
+
+.top-bar {
+  background: $topbar-bg-color;
+
+  .row {
+    background-color: $topbar-bg-color;
+  }
+
+  .name h1 {
+    color: #FFFFFF;
```

Comment:
Color literals like `#FFFFFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFFFFF` should be written as `#ffffff`